### ISSUE TITLE
Python Interface on Windows

### ISSUE THREAD

**Issue by eriq-augustine** [eriq-augustine/psl#203]
There seems to be some issues with the Python interface on Windows.

Reports are that it stems from differences in the asyncio implementation on Windows.

**Comment by eriq-augustine** [comment]
Reports of this error:
```
 File "C:\python-3.7\lib\asyncio\base_events.py", line 463, in _make_subprocess_transport
    raise NotImplementedError
```

**Comment by eriq-augustine** [comment]
I think this should be fixed since we moved away from asyncio in cdcf4cd8776ff21121a1a5111be21d158fb20e22.

We don't have any Windows CI, but we will close this for now as we stuck to only platform-safe parts of subprocess.
